src/frontend/src/components/LoginContainer/Form/Form.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+// import './Form.css';
+import Button from './Button';
+
+const Form: React.FC = () => {
+  return (
+    <div className="Form">
+      <form className="Login">
+        <div className="form-group">
+          <p> Sign In </p>
+          <div className="e-mail">
+            <p className="info">E-Mail</p>
+            <input type="text" name="e-mail" />
+          </div>
+          <div className="password">
+            <p className="info">Password</p>
```

Comment:
If you are not using CSS, don't prematurely use CSS classes that may not be correct anymore/in the future.